userguide/getting-started/quick_start_with_kb_api.adoc

```diff
@@ -201,9 +213,15 @@ $account = $accountApi->createAccount($accountData, 'tutorial', NULL, NULL);
 </div>
 ++++
 
```

Comment:
Wondering if we should keep the Go and PHP code samples or not since we do not include code samples for these in our slate docs. @pierre, @sbrossie any thoughts?

Reply:
Could we update our Slate docs to have examples for all languages supported (Java, Ruby, Python, Go, JS, and PHP)? 😇 Maybe we can divide and conquer depending on the skillset in the team?

But I appreciate that's a long term effort... I'd still keep the various examples in the docs in the meantime, even if Slate is incomplete.